ods_tools/oed/exposure.py

```diff
@@ -229,3 +244,96 @@ def check(self, validation_config=None):
             validation_config = self.validation_config
         validator = Validator(self)
         return validator(validation_config)
+
+    def to_version(self, version):
+        """
+        goes through location to convert columns to specific version.
+        Right now it works for OccupancyCode and ConstructionCode.
+        (please note that it only supports minor version changes in "major.minor" format, e.g. 3.2, 7.4, etc.)
+
+        Args:
+            version (str): specific version to roll to
+
+        Returns:
+            itself (OedExposure): updated object
+        """
+
+        def version_to_tuple(version_str):
+            """Converts the version in string format to a tuple of integers"""
+            return tuple(map(int, version_str.split('.')))
```

Comment:
i think
return tuple(map(int, version_str.split('.', 2)[:2]))
is better because you could have a version name
3.1.0rc